### DISCUSSION TITLE
Show provenance checkmark in versions list

### DISCUSSION THREAD

**Discussion by remcohaszing** [npm/feedback#915] [Registry] [answered]
It could be useful to show the provenance checkmark next to the version in the list of package versions. This way the user could immediately see something is off when a version without provenance is released for a package that normally does provide provenance.

I.e. https://www.npmjs.com/package/estree-util-value-to-estree?activeTab=versions could look like this:

![image](https://user-images.githubusercontent.com/779047/233963244-0257a027-2b55-4cac-a69c-b404117e41f8.png)

**Comment by feelepxyz** [comment]
@remcohaszing 💯  definitely something we want to do and planning, including highlighting this in the search results.

**Reply by remcohaszing** [reply]
Nice! The more checkmarks the merrier ✔️

**Comment by feelepxyz** [comment, marked as answer]
@remcohaszing this is now done: https://www.npmjs.com/package/sigstore?activeTab=versions ✅
